### PR TITLE
Add tisysconfig package

### PR DESCRIPTION
**Pull request by JFreitas9**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JFreitas9** [comment]
Hello @conda-forge/staged-recipes please review my pull request.

File: recipes/sysconfig/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  number: {{ build }}
+
+requirements:
+  run:
+    - nodejs
+
+about:
+  home: https://www.ti.com/tool/SYSCONFIG
+  summary: The SysConfig tool is used configure TI software and hardware
+  license: None
```
Comment (JFreitas9) [inline recipes/sysconfig/meta.yaml:32]: I can't find a proper license to add here. Can you help me?

**Comment by JFreitas9** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes

> Hello @conda-forge/staged-recipes please review my pull request.

**Comment by JFreitas9** [comment]
@conda-forge/staged-recipes please review my pull request.
Thanks in advance!

**Review by carterbox — CHANGES_REQUESTED** [review]
Two issues:

1. License. No, we cannot find the license for you. As the maintainer of this package you should be able to find the license. This software must have a license which allows modification and redistribution of the source otherwise, we cannot host this package. Ask the author for the license.

2. conda-force packages must be build from source files using our compilers. They cannot be created using an installer (which is what is being done here) because that usually means repackaging of pre-compiled binaries.

**Comment by carterbox** [comment]
Closing because there is no evidence that this program is licensed appropriately.

**Pull request closed without merging**